images/build-e2e/Containerfile

```diff
@@ -1,5 +1,5 @@
 
-FROM registry.access.redhat.com/ubi8/go-toolset:1.20 AS builder
+FROM registry.access.redhat.com/ubi8/go-toolset:1.21 AS builder
```

Comment:
can you also update https://github.com/crc-org/crc/blob/main/update-go-version.sh and may be add another like for `Containerfile` ?

Reply:
Hey can you add this? or do you prefer create another PR, I want to merge this one as soon as possible

Reply:
ahh, sorry i forgot about this, updating now!

Reply:
updated now!